### ISSUE TITLE
Tinkers' Construct cuts out extrautils prefix from localisation.

### ISSUE THREAD

**Issue by Psithief** [rwtema/extrautilities_old_issues#791]
Versions
- Tinkers' Construct 1.7.10-1.8.8
- Extra Utilities 1.2.12
- Forge 1.7.10-10.13.4.1558

---

```
material.extrautils.bedrockium=Bedrockium
material.extrautils.bedrockium.ability=Heavy
material.extrautils.magicwood=Magical Wooden
material.extrautils.magicwood.ability=Modifiable
material.extrautils.unstableIngot=Unstable Induced
material.extrautils.unstableIngot.ability=Mathematical!
```

leading to this in-game

![2016-01-26_22 06 10 - copy](https://cloud.githubusercontent.com/assets/1124846/12583413/a62badb6-c47d-11e5-9726-d5783933e291.png)

TC seems to be making its own localisation identifier so it looks for `material.bedrockium`, `material.magicwood`, etc.

**Comment by Blue64** [comment]
this explains a few double fluid entries in my NEI, thanks for that. did you report it to TiCon?

**Comment by Psithief** [comment]
No, they moved on to a different version and no longer were providing any support.

**Comment by Blue64** [comment]
one workaround would be to use [MineTweaker](https://minecraft.curseforge.com/projects/minetweaker3/files) ([1.7.10](https://minecraft.curseforge.com/projects/minetweaker3/files?filter-game-version=2020709689%3A4449)) & [ModTweaker](https://minecraft.curseforge.com/projects/modtweaker/files) ([1.7.10](https://minecraft.curseforge.com/projects/modtweaker/files?filter-game-version=2020709689%3A4449)) to direct any Recipes to/from the correct Fluids, such as the Bedrockium Fluid which I believe gets registered twice in [my ModPack](https://github.com/Blue64/Infernal_Rift) (WiP), not sure if that's caused by multiple Mods registering it, or if ExU used different formatting on some things (perhaps he did a prefix on some and not others).
